frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -32,6 +32,10 @@ const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({
     (e) => e.getCustomPropertiesMap().get('task_name')?.getStringValue() === task.name,
   );
 
+  const isCachedTask =
```

Comment:
Seems like this could be simplified to 
```
const isCachedTask = !!taskExecution?.getCustomPropertiesMap().get('cached_execution_id')?.getStringValue();
```

Reply:
i will add the change when i fix the tests